src/index.js

```diff
@@ -1,50 +1,65 @@
-import Autocomplete from './components/Autocomplete/Autocomplete';
+import { buildStatefulComponent } from './util/state-management';
```

Comment:
this kinda sucks and i'm halfway tempted to change this file to a common js export syntax so we can dynamically generate the exports... at least until we have some static analysis use-case for using es6 exports